modules/operators/src/map-response.ts

```diff
@@ -6,6 +6,31 @@ type MapResponseObserver<T, E, R1, R2> = {
   error: (error: E) => R2;
 };
 
+/**
+ * `mapResponse` is a map operator with included error handling.
+ * It is similar to `tapResponse`, but allows to map the response as well.
+ *
+ * The main use case is for NgRx Effects which requires an action to be dispatched.
+ *
+ * @usageNotes
+ * ```ts
+ * export const loadAllUsers = createEffect((
+ *   actions$ = inject(Actions),
+ *   usersService = inject(UsersService)
+ * ) => {
+ *   return actions$.pipe(
+ *     ofType(UsersPageActions.opened),
+ *     exhaustMap(() => {
+ *       return usersService.getAll().pipe(
+ *         mapResponse({
+ *           next: (users) => UsersApiActions.usersLoadedSuccess({ users }),
+ *           error: (error) => UsersApiActions.usersLoadedFailure({ error }),
+ *         })
+ *       );
+ *     })
+ *   );
+ * });
```

Comment:
The markdown brackets should be closed.

```suggestion
 * ```ts
 * export const loadAllUsers = createEffect((
 *   actions$ = inject(Actions),
 *   usersService = inject(UsersService)
 * ) => {
 *   return actions$.pipe(
 *     ofType(UsersPageActions.opened),
 *     exhaustMap(() => {
 *       return usersService.getAll().pipe(
 *         mapResponse({
 *           next: (users) => UsersApiActions.usersLoadedSuccess({ users }),
 *           error: (error) => UsersApiActions.usersLoadedFailure({ error }),
 *         })
 *       );
 *     })
 *   );
 * });
 * ```
```